DownFall/Views/BackpackView.swift

```diff
@@ -16,79 +16,226 @@ import SpriteKit
 
 class BackpackView: SKSpriteNode {
     
-    let playableRect: CGRect
-    var itemArea: SKSpriteNode?
-    var descriptionArea: SKSpriteNode?
-    var ctaButton: SKShapeNode?
-    var abilities: [AnyAbility] = []
+    let viewModel: TargetingViewModel
+    
+    weak var touchDelegate: Renderer?
+    
+    
+    private var tileSize: CGFloat
+    private var boardSize: CGFloat
+    
+    private var toastMessageContainer: SKShapeNode?
+    private let playableRect: CGRect
+    private var background: SKSpriteNode
+    private var targetingArea: SKSpriteNode
+    private var viewContainer: SKSpriteNode
+    
+    private var inventoryArea: SKSpriteNode
+    
+    private var itemArea: SKSpriteNode?
+    private var descriptionArea: SKSpriteNode?
+    private var ctaButton: Button?
+    private var cancelButton: Button?
+    private var nextItemsButton: Button?
```

Comment:
I dont these have to be optional.

Reply:
made some lazy cuz they needed self as a deleagte

moved the others up in the init